pytest_fixtures/component/taxonomy.py

```diff
@@ -124,47 +137,71 @@ def module_gt_manifest_org(module_target_sat):
 @pytest.fixture(scope='session')
 def session_entitlement_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='session')
 def session_sca_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.golden_ticket) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='module')
 def module_entitlement_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
         yield manifest
 
 
+@pytest.fixture(scope='module')
+def module_extra_rhel_entitlement_manifest():
+    """Yields a manifest in entitlementmode with subscriptions determined by the
+    'manifest_category.extra_rhel_entitlement` setting in conf/manifest.yaml."""
+    with Manifester(manifest_category=settings.manifest.extra_rhel_entitlement) as manifest:
+        yield manifest
+
+
 @pytest.fixture(scope='module')
 def module_sca_manifest():
     """Yields a manifest in Simple Content Access mode with subscriptions determined by the
-    `manifest_category.golden_ticket` setting in manifester_settings.yaml."""
+    `manifest_category.golden_ticket` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.golden_ticket) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='function')
 def function_entitlement_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
         yield manifest
 
 
+@pytest.fixture(scope='function')
+def duplicate_entitlement_manifest():
```

Comment:
```suggestion
def entitlement_manifest():
```
OR
```suggestion
def function_entitlement_manifest():
```

Reply:
Simple Why should we have `duplicate` functions or fixtures. The name duplicate is really confusing, btw if these or the other module functions are uploading two manifest then lets name them `double_entitlement_manifest` or something ?

Reply:
Looking back through the tests, I'm only seeing one that requires uploading a manifest to two separate organizations, and I actually wrote a local fixture in that module that provides the second manifest. So maybe the best thing to do here is actually remove these fixtures from `taxonomy.py`. That way we avoid the confusion with the name and don't have global fixtures defined that are only being used in a single test.

Reply:
Agree!